### DISCUSSION TITLE
Unable to resolve type from Nuget package

### DISCUSSION THREAD

**Discussion by martinrhan** [AvaloniaUI/Avalonia#10349] [Q&A]
Steps to reproduce:
1. Create a new project using "Avaolonia MVVM Appilcation" template.
2. Add package reference ``<PackageReference Include="NP.Avalonia.UniDock" Version="1.0.48" />``
3. In MainWindow.axaml, add the namespace`` xmlns:np="https://np.com/visuals"``
4. Add attached property to the window as well `` np:DockAttachedProperties.DockChildWindowOwner="{Binding RelativeSource={RelativeSource Mode=Self}}"``
5. Build.
6. Build successful, but there is an error, telling "Unable to resolve type DockAttachedProperties"

Same issue posted at: https://github.com/npolyak/NP.Avalonia.UniDock/issues/18

**Comment by maxkatz6** [comment]
I assume it's only a previewer error. Which was reported here https://github.com/AvaloniaUI/Avalonia/issues/9574
It was fixed in the 11.0 previews. But there are workarounds for the 0.10 https://github.com/AvaloniaUI/Avalonia/issues/9574#issuecomment-1332258588.
